unit_testing/hello_test.cpp

```diff
@@ -0,0 +1,9 @@
+#include <gtest/gtest.h>
+
+// Demonstrate some basic assertions.
+TEST(HelloTest, BasicAssertions) {
```

Comment:
As gtest now seems to be working well, let's remove this before merging.

Reply:
Good move